pydantic_ai_slim/pydantic_ai/agent.py

```diff
@@ -20,7 +20,7 @@
     models,
     result,
 )
-from .messages import TextPart, ToolCallPart
```

Comment:
we already have `_messages` imported, to avoid cluttering the namespace excessively, we should just use that.

Reply:
Ah oops, autoimport mistake. Thanks for catching.